### PR TITLE
Closed

### PR DESCRIPTION
**Pull request by 8bitmp3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ comes with everything you need to start your research, including:
 
 ## Quick install
 
-You will need Python 3.6 or later and a working [JAX](https://github.com/google/jax/blob/main/README.md)
```
Comment (8bitmp3) [inline README.md:67]: Perhaps:

"...3.7 and above, and a working JAX installation..."

or

"...3.7-3.10, and a working JAX installation..."

@IvyZX @cgarciae

**Comment by cgarciae** [comment]
Curious about the name of this PR, doesn't seem to be related with the code changes (unless I am missing something).

**Comment by chiamp** [comment]
> Curious about the name of this PR, doesn't seem to be related with the code changes (unless I am missing something).

Yes, the diff looks identical to #2625

**Comment by 8bitmp3** [comment]
@chiamp @cgarciae @IvyZX The git rebase reimported #2625 by @chiamp . Here is the main diff https://github.com/google/flax/pull/2683/commits/a71b5c7e1f8853e0697d738e73dcb7d91c5ae5b7

**Comment by 8bitmp3** [comment]
After syncing the branch with `main`, I still have your commit @chiamp - not sure how to remove it 😅 

image

**Comment by 8bitmp3** [comment]
Related PR https://github.com/google/flax/pull/2682/files by @cgarciae

**Comment by cgarciae** [comment]
Seems like there are a couple of doc improvements mixed in this PR? Is this intentional?

**Comment by marcvanzee** [comment]
@8bitmp3 this PR has a conflict, could you please resolve it?

**Pull request closed without merging**